recurser/string-is
# Improve the regex modifer interface

## How to test the PR

1. Edit the regex string to add modifiers manually - it should update the modifier dropdown
2. Change the modifier dropdown - it should update the regex string.
